packages/framework/src/Twig/ImageExtension.php

```diff
@@ -47,21 +41,18 @@ class ImageExtension extends Twig_Extension
 
     /**
      * @param string $frontDesignImageUrlPrefix
-     * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
      * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
      * @param \Shopsys\FrameworkBundle\Component\Image\ImageLocator $imageLocator
      * @param \Shopsys\FrameworkBundle\Component\Image\ImageFacade $imageFacade
```

Comment:
Btw, these annotations are really weird. Do you still see value in them? :)

They are also often missed, as I noticed on many places even only in this PR, which makes code look randomly developed.